stdlib/public/core/StringNormalization.swift

```diff
@@ -247,47 +246,45 @@ internal func transcodeToUTF8(
   return (readIndex, writeIndex)
 }
 
-internal func fastNormalize(
-  readIndex: String.Index,
-  guts: _StringGuts,
-  outputBuffer: UnsafeMutableBufferPointer<UInt8>,
-  icuInputBuffer: UnsafeMutableBufferPointer<UInt16>,
-  icuOutputBuffer: UnsafeMutableBufferPointer<UInt16>
+internal func _fastNormalize(
+  _ readIndex: String.Index,
+  _ sourceBuffer: UnsafeBufferPointer<UInt8>,
+  _ outputBuffer: UnsafeMutableBufferPointer<UInt8>,
+  _ icuInputBuffer: UnsafeMutableBufferPointer<UInt16>,
+  _ icuOutputBuffer: UnsafeMutableBufferPointer<UInt16>
```

Comment:
I think argument labels are necessary here, otherwise it's hard to know what buffer to pass where.

E.g. `_fastNormalized(startingAt:from:into:icuInputBuffer:icuOutputBuffer:)`

Reply:
ok